test/languages.ts

```diff
@@ -9,7 +9,8 @@ export type LanguageFeature =
   | "date-time"
   | "integer-string"
   | "bool-string"
-  | "uuid";
+  | "uuid"
+  | "minmax";
```

Comment:
You'll have to add `minmaxlength` and `pattern`, too.

Reply:
Strange, all three have executed, although I have zero idea, whether the negative tests were executed at all :)

Reply:
My issue was exactly that when I manually executed the three tests from commandline I thought that the .fail part would also be executed, but I guess this is not the case, right

Reply:
`something.1.fail.feature.json` will only run if the language supports `feature`.

Reply:
Ah understood, thanks for clarifying it